messages/en.json

```diff
@@ -45,7 +46,9 @@
         "Despite protests, the mob drags their victim to the hanging tree. \u0002{0}\u0002 succumbs to the will of the horde, and is hanged. The villagers have killed a{1} \u0002{2}\u0002.",
         "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows. Once the twitching stops, it is discovered that the village lynched a{1} \u0002{2}\u0002.",
         "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, pulls the pin on a grenade. They hesitate, and it explodes, killing them.",
-        "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, throws a grenade at the mob. The grenade explodes early."
+        "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, throws a grenade at the mob. The grenade explodes early.",
+        "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered in chunks of dead \u0002{2}\u0002.",
```

Comment:
"With the remains of a late __seer__" for example, or "With the remains of the late comrade"